### PR TITLE
[REVIEW] First pass of jit based its timestamp refactoring

### PR DESCRIPTION
**Pull request by thomcom**
Convert its timestamp correctly to np datetime64 format.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -51,7 +51,7 @@
 - PR #68 Use YYMMDD tag in nightly build of cuspatial python
 - PR #97 Drop `cython` from run requirements
 - PR #82 Added update-version.sh
-- PR #86 Add Shapefile reader for polygons
```
Comment (raydouglass) [inline CHANGELOG.md]: I think a weird merge happened. Or looks like you switched the target branch. You might want to merge branch-0.12 into your branch or maybe just rebase.

**Comment by harrism** [comment]
I keep wondering what "its" is. Can you just use the new `cudf::timestamp` types?  They are standards-based (C++ std::chrono)

**Comment by harrism** [comment]
@thomcom is this PR still valid/needed?

**Comment by harrism** [comment]
If it's just useful for conversion, why not convert them into the new format and then get rid of the old data files and functionality to load them?

**Comment by harrism** [comment]
Moving to 0.15.

**Comment by harrism** [comment]
Closing. Please reopen if needed.

**Pull request closed without merging**